app/models/form/sales/questions/deposit_value_check.rb

```diff
@@ -0,0 +1,25 @@
+class Form::Sales::Questions::DepositValueCheck < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "deposit_value_check"
+    @check_answer_label = "Deposit confirmation"
+    @header = "Are you sure that the deposit is this much higher than the buyer's savings?"
```

Comment:
"this much higher" seems weird phrasing to me - shouldn't we be showing the user how much higher it is?

Reply:
Yup! I agree, I think all of the validation messages will be rewritten at some point and this is just a placeholder